runs.py

```diff
@@ -21,12 +22,17 @@
 from logger import logger
 from events import add_listener
 from utils import convert_class_to_obj, get_req_data
+from youtubearchive import YoutubeArchive, YoutubeVideoMetadata
 
 __all__ = ["get_latest_run"]
 
 _cache: dict[str, RunParser] = {}
 _ts_cache: dict[int, RunParser] = {}
 
+# TODO: Should this be made here?
```

Comment:
@Vgr255 Sorry I am a python rookie still, how do we do instantiate this sort of thing?

Reply:
This should be made in a `setup_init` event listener, and set to `None` globally.